docs/r2/tensorboard_quickstart.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>Not sure what's preferred, but you can just do</p><p><br></p><p>activation='relu' and</p><p>activation='softmax'</p><p><br></p><p>I find it easier to read.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1870/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>String activations are preferred and it becomes consistent across all the tutorials.</p>